File: NDPCommon/src/main/java/com/slender/utils/RandomToolkit.java
package com.slender.utils;

import java.util.random.RandomGenerator;

public final class RandomToolkit {
    private RandomToolkit() {}

    private static final RandomGenerator generator = RandomGenerator.of("L64X128MixRandom");

    public static int getRandomInt(final int min ,final int max) {
        return generator.nextInt(max-min+1)+min;
    }

    public static int getRandomExpireTime(){
        return getRandomInt(-TimeToolkit.Unit.MINUTE*2,TimeToolkit.Unit.MINUTE*2);
    }
}

File: NDPCommon/src/main/java/com/slender/utils/RandomToolkit.kt
package com.slender.utils

import java.util.random.RandomGenerator

object RandomToolkit {
    private val generator: RandomGenerator = RandomGenerator.of("L64X128MixRandom")

    fun getRandomInt(min: Int, max: Int): Int
        = generator.nextInt(max - min + 1) + min

    val randomExpireTime: Long
        get() = getRandomInt(-TimeToolkit.Unit.MINUTE * 2, TimeToolkit.Unit.MINUTE * 2).toLong()
}
